pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1280,8 +1280,8 @@ SELECT description
 					}
 					return nonUpdatableEvents, err
 				}
-				tableDesc, ok := table.(catalog.TableDescriptor)
-				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
```

Comment:
removing the check on `ok` is fine because `GetImmutableTableInterfaceByID` looks up the descriptor with the Required flag.